paper/paper.md

```diff
@@ -0,0 +1,210 @@
+---
+title: BurnMan -- a Python toolkit for planetary geophysics, geochemistry and thermodynamics'
+tags:
+  - Python
+  - planetary
+  - geophysics
+  - geochemistry
+  - thermodynamics
+authors:
+  - name: Robert Myhill
+    orcid: 0000-0001-9489-5236
+    corresponding: true
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Sanne Cottaar
+    orcid: 0000-0003-0493-6570
+    affiliation: 2
+  - name: Timo Heister
+    orcid: 0000-0002-8137-3903
+    affiliation: 3
+  - name: Ian Rose
+    affiliation: 4
+  - name: Cayman Unterborn
+    affiliation: 5
+  - name: Juliane Dannberg
+    orcid: 0000-0003-0357-7115
+    affiliation: 6
+  - name: Rene Gassmoeller
+    orcid: 0000-0001-7098-8198
+    affiliation: 6
+affiliations:
+ - name: University of Bristol, UK
+   index: 1
+ - name: University of Cambridge, UK
+   index: 2
+ - name: University of Utah, USA
+   index: 3
+ - name: Independent Researcher, USA
+   index: 4
+ - name: Southwest Research Institute, USA
+   index: 5
+ - name: University of Florida, USA
+   index: 6
+date: 13 September 2022
+bibliography: paper.bib
+
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+# aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+# aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+
+# Summary
+Many branches of physics, chemistry and Earth sciences build
+complex materials from simpler constructs: rocks are composites
+made up of one or more phases; phases are solutions of more than one
+endmember and endmembers are usually mixtures of more than one element.
+The properties of the endmember building blocks at different pressures and
+temperatures can be provided by a wide array of different equations of state.
+The averaging of the endmember properties within solutions and composite
+materials can also be achieved in several different ways.
+
+# Statement of need
+Earth Scientists are interested in a number of different material properties,
+including seismic velocities, heat capacities and density as a function of
+pressure and temperature. Many of these properties are connected to each
+other by physical laws. Building models of individual phases to compute
+these properties and checking them can be time-consuming and prone to error,
+so it is desirable to have well-tested and benchmarked software that
+provides convenient functions to calculate the properties of complex
+composite materials, and to fit new models.
+
+`BurnMan` is an open source, extensible mineral physics toolbox designed to
+calculate the physical properties of natural materials
+within the Earth and other planets. The toolbox has a class-based, modular
+design that allows users to calculate many low-level properties that
+are not accessible using existing codes, and to combine various tools in
+novel, creative ways.
```

Comment:
I think this paragraph should be part of the summary above (as a second paragraph). According to the JOSS website the summary should:
```
A summary describing the high-level functionality and purpose of the software for a diverse, non-specialist audience.
```
Which is exactly the contrent of this paragraph.

I think the rest of the text in the problem statement remains in good shape even if this paragraph is moved up.

Reply:
Thanks, I agree :)